src/window/User Guide.md

```diff
@@ -0,0 +1,132 @@
+% User Guide to Using Piston Windows
+
+# Guide to using Piston Windows
+
+## Getting Help
+
+*Notice! Piston is being actively developed. Please let us know if anything in this tutorial is outdated.*
+
+See the main [Piston Guide](../../GUIDE.md) for general help with Piston.
+
+## Overview
+
+The piston engine is designed to be an abstraction. It provides the same functionality through several different backends, each of which have their own quirks and extra features.
+
+Because of this abstraction, some of the details of the piston API may seem unnecessary or useless. The purpose of this guide is to give concrete guidelines on the creation and usage of piston windows.
+
+If you mostly know what you're doing, but need a quick refresher, see the quick reference at the bottom of this guide.
+
+# Windows
+
+## Choosing a Backend
+
+There are a few available backends for piston, and it's important to choose one for your project.
+For the start of most projects, any backend will work. If you're coming from another programming language like C or C++, you should use a backend that you've worked with in the past. Beyond these superficial differences, there are other things to consider:
```

Comment:
"should" => "can" otherwise I don't think the transition to Glutin is particularly hard for C or C++ programmers.

"you have"